### ISSUE TITLE
Pls don't use blue for text 

### ISSUE THREAD

**Issue by ncoleman** [segler-alex/RadioDroid#1228]
For people with short sight, or older people, (dark) blue is the worst colour choice for text.  The reason is that, looking at red at one end and blue at the other end of the visual spectrum, blue's frequency is the most in error, causing the text to blur.

Plus, on a dark background, there is little contrast between the dark blue text and the dark background.

Even with my glasses on I cannot make out the text at the bottom showing the station that is playing unless in bright light.  No chance if I'm looking at my phone at night.

**Comment by ZTetriminos** [comment]
Yes, please. This is barely readable. Most of the time I forget where the play/pause button is.

Thank you!

![Screenshot_20240906-083557_1](https://github.com/user-attachments/assets/ad8e393a-ef3f-4f2e-82ec-82756b2058eb)
